cirq/google/engine/engine.py

```diff
@@ -360,6 +362,65 @@ def run_batch(
                                         description=job_description,
                                         labels=job_labels)
 
+    def run_calibration(
+            self,
+            layers: List['cirq.google.CalibrationLayer'],
+            program_id: Optional[str] = None,
+            job_id: Optional[str] = None,
+            processor_ids: Sequence[str] = (),
+            gate_set: Optional[sgs.SerializableGateSet] = None,
+            program_description: Optional[str] = None,
+            program_labels: Optional[Dict[str, str]] = None,
+            job_description: Optional[str] = None,
+            job_labels: Optional[Dict[str, str]] = None,
+    ) -> engine_job.EngineJob:
+        """Runs the supplied Circuits via Quantum Engine.Creates
+
+        This will combine each Circuit provided in `programs` into
+        a BatchProgram.  Each circuit will pair with the associated
+        parameter sweep provided in the `params_list`.  The number of
+        programs is required to match the number of sweeps.
+
+        This method does not block until a result is returned.  However,
+        no results will be available until the entire batch is complete.
+
+        Args:
+            programs: The Circuits to execute as a batch.
+            program_id: A user-provided identifier for the program. This must
+                be unique within the Google Cloud project being used. If this
+                parameter is not provided, a random id of the format
+                'prog-################YYMMDD' will be generated, where # is
+                alphanumeric and YYMMDD is the current year, month, and day.
+            job_id: Job identifier to use. If this is not provided, a random id
+                of the format 'job-################YYMMDD' will be generated,
+                where # is alphanumeric and YYMMDD is the current year, month,
+                and day.
+            processor_ids: The engine processors that should be candidates
```

Comment:
Is there any use case to specify multiple processors here?

Reply:
I don't really think there is ever a case to specify multiple processors, but this is consistent with the other usage.

Actually, I am thinking of changing this to support one or more processors for all the calls.  What do you think?

Reply:
That's true, but I think we should keep this interface consistent with run and run_batch for now.  If we are going to change this (which would be breaking), we should do it together.

Reply:
run_calibration() now supports both processor_ids and processor_id